wagtail/tests/settings.py

```diff
@@ -91,12 +91,12 @@
     'wagtail.tests.snippets',
     'wagtail.tests.routablepage',
     'wagtail.tests.search',
-    'wagtail.contrib.wagtailstyleguide',
-    'wagtail.contrib.wagtailsitemaps',
-    'wagtail.contrib.wagtailroutablepage',
-    'wagtail.contrib.wagtailfrontendcache',
-    'wagtail.contrib.wagtailapi',
-    'wagtail.contrib.wagtailsearchpromotions',
+    'wagtail.contrib.styleguide',
+    'wagtail.contrib.sitemaps',
+    'wagtail.contrib.routablepage',
+    'wagtail.contrib.frontendcache',
+    'wagtail.contrib.api',
+    'wagtail.contrib.searchpromotions',
```

Comment:
Couldn’t it be the occasion to be more pythonic and clear by adding underscores between words?
The modules would become:
- `style_guide`
- `sitemaps` (special case here since Sitemaps is a technology name)
- `routable_page`
- `frontend_cache`
- `api`
- `search_promotions`

If others think this is the right way to go, then an additional pull request should be done to rename `modeladmin` to `model_admin`. This pull request could also be updated to do that, but I fear it will be overly complex.